zipline/assets/assets.py

```diff
@@ -678,3 +730,113 @@ class AssetConvertible(with_metaclass(ABCMeta)):
 
 class NotAssetConvertible(ValueError):
     pass
+
+
+class AssetFinderCachedEquities(AssetFinder):
```

Comment:
Oh you know what - I totally missed the word Equities in the name as it currently stands.  I think it's a good name as is.